samples/core/secret/secret.py

```diff
@@ -57,11 +56,8 @@ def use_gcp_api_op():
 def secret_op_pipeline(url='gs://ml-pipeline-playground/shakespeare1.txt'):
   """A pipeline that uses secret to access cloud hosted resouces."""
 
-  gcs_read_task = gcs_read_op(url).apply(
```

Comment:
The use of secrets seems to be the core feature demonstrated by this sample. Perhaps this should not be removed.

Reply:
Yes, you are right. Current situation is that we are not running tests that use a service account token. Shall we remove this sample from presubmit/postsubmit tests?